js/blogtini.js

```diff
@@ -151,22 +151,33 @@ jekyll (GitHub Pages) plugins:
 /* eslint-disable no-continue */
 import yml from 'https://esm.archive.org/js-yaml'
 import dayjs from 'https://esm.archive.org/dayjs'
-import showdown from 'https://esm.archive.org/showdown'
```

Comment:
I have something about that

Reply:
interesting.  i've found showdown quite excellent..

Reply:
It is quite excellent.

I'm going to make a customElement that'll use it. The way of HTMLizing, I have some ideas. Some partly started on my detached fork (renoirb/blogtini). It's broken ATM, but I'm trying to simplify and separate concerns so that it'll be easier to make our own theme.

Say the engine uses blogtini-article, has a few slots, the default takes the markdown text with front matter (another customElement).  Then, the person who's using blogtini can adapt their version of blogtini-article.

someting like that.